lib/render.go

```diff
@@ -41,3 +64,33 @@ func Markdown(source string) string {
 
 	return buf.String()
 }
+
+type preWrapStruct struct {
+}
+
+const start = `
+<div class="highlight">
+<pre %s>
+<code>`
+
+func (p *preWrapStruct) Start(code bool, styleAttr string) string {
+	w := &strings.Builder{}
+
+	styleAttr = strings.TrimSpace(styleAttr) // this param has spaces sometimes
+
+	if strings.HasPrefix(styleAttr, `style="`) {
+		style1 := styleAttr[:len(styleAttr)-1] //remove the trailing quote
+		style2 := `;overflow-x: auto"`
+		newStyle := style1 + style2
```

Comment:
If you don't mind, I'd prefer to see it like this:

```
newStyle := styleAttr[:len(styleAttr)-1] + `;overflow-x: auto"`
```

lesser amount of variables, less naming, etc. I think it's clear enough. :) One thing I'm not sure if styleAttr doesn't start with `style=` (when it's empty or what?), don't you want to just output `style="overflow-x: auto"`?

Reply:
Good idea! 
I also like the idea of overflow-x doesn't match our expectations. I suspect that branch can never be hit, but I figured maybe we should handle it just in case.